<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modif_med</class>
 <widget class="QDialog" name="modif_med">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:white;
background:#001328;
</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>160</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="txtNommed">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;background-color: WHITE;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtTaux">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;background-color: WHITE;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblnummede">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">COLOR:white;</string>
   </property>
   <property name="text">
    <string>N°medecin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_connect">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>FZFGERG</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblNommed">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">COLOR:white;</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblTaux">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">COLOR:white;</string>
      </property>
      <property name="text">
       <string>Taux_journalier</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>hgqe</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nummed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">COLOR:white;</string>
   </property>
   <property name="text">
    <string>N°medecin</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;background-color: WHITE;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtnummed">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;background-color: WHITE;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #5E64A6;
color:white;</string>
   </property>
   <property name="text">
    <string>Recherche</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #5E64A6;
color:white;</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
